R/zchunk_LA154.transportation_UCD.R

```diff
@@ -40,120 +44,390 @@ module_energy_LA154.transportation_UCD_DISABLED <- function(command, ...) {
-    # There's also a `same_precursors_as(x)` you can use
-    # If no precursors (very rare) don't call `add_precursor` at all
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    IEA_hist_data_times_UCD_shares_region_scaled_to_enduse_data %>%
+      add_title("Regional transportation energy data at UCD transportation technology level") %>%
+      add_units("EJ") %>%
+      add_comments("Aggregated country-level transportation energy data to UCD transportation technologies") %>%
+      add_comments("Scaled to transport end-use data") %>%
       add_legacy_name("L154.in_EJ_R_trn_m_sz_tech_F_Yh") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
+      add_precursors("common/iso_GCAM_regID", "temp-data-inject/L101.in_EJ_ctry_trn_Fi_Yh",
+                     "temp-data-inject/L1011.in_EJ_ctry_intlship_TOT_Yh", "L131.in_EJ_R_Senduse_F_Yh",
+                     "energy/mappings/calibrated_techs_trn_agg", "energy/enduse_fuel_aggregation",
+                     "energy/mappings/UCD_ctry","energy/mappings/UCD_techs",
+                     paste0("energy/UCD_trn_data_", energy.TRN_SSP)) %>%
       add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
       L154.in_EJ_R_trn_m_sz_tech_F_Yh
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+
+    IEA_hist_data_times_UCD_shares %>%
+      add_title("Country-level transportation energy data at UCD transportation technology level") %>%
+      add_units("EJ") %>%
+      add_comments("Aggregated country-level transportation energy data to UCD transportation technologies") %>%
       add_legacy_name("L154.in_EJ_ctry_trn_m_sz_tech_F") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
+      add_precursors("common/iso_GCAM_regID", "temp-data-inject/L101.in_EJ_ctry_trn_Fi_Yh",
+                     paste0("energy/UCD_trn_data_", energy.TRN_SSP), "energy/mappings/calibrated_techs_trn_agg",
+                     "energy/mappings/UCD_ctry", "energy/mappings/UCD_techs",
+                     "temp-data-inject/L1011.in_EJ_ctry_intlship_TOT_Yh") %>%
       add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
       L154.in_EJ_ctry_trn_m_sz_tech_F
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+
+    out_var_df[["intensity_MJvkm"]] %>%
+      add_title("Transportation energy intensity") %>%
+      add_units("MJ / vkm") %>%
```

Comment:
This is not a huge deal, but I think of units, whether its MJ or MJ/vkm, as a single item or description and not an operation.